src/main/kotlin/com/ampnet/identityservice/service/impl/UserServiceImpl.kt

```diff
@@ -45,6 +51,19 @@ class UserServiceImpl(
             userRepository.save(User(address))
         }
 
+    @Transactional
+    override fun verifyUserWithUserInfo(request: KycTestRequest): UserWithInfo {
+        val user = getUser(request.address)
+        val userInfo = com.ampnet.identityservice.persistence.model.UserInfo(
```

Comment:
` com.ampnet.identityservice.persistence.model.UserInfo` why not just `UserInfo`? It's imported.